barretenberg/cpp/src/barretenberg/honk/transcript/transcript.hpp

```diff
@@ -79,24 +79,35 @@ template <typename FF> class BaseTranscript {
     TranscriptManifest manifest;
 
     /**
-     * @brief Compute c_next = H( Compress(c_prev || round_buffer) )
-     *
+     * @brief Compute next challenge c_next = H( Compress(c_prev || round_buffer) )
+     * @details This function computes a new challenge for the current round using the previous challenge
+     *          and the current round data, if they are exist.
```

Comment:
Weird formatting

Reply:
I assumed that it should be aligned with the first line, but I'll align it with the *s.